src/app/list/list.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component} from '@angular/core';
+import {Store} from "@ngrx/store";
+import * as TaskActions from "./state/list/list.actions";
+import {Observable} from "rxjs";
+import {selectTasks} from "./state/list/list.selectors";
+import {Task} from "./state/list/list.state";
+
+@Component({
+  selector: 'app-list',
+  templateUrl: './list.component.html',
+  styleUrls: ['./list.component.css']
+})
+export class ListComponent {
+  list$: Observable<Task[]> = this.store.select(selectTasks);
+
+  constructor(private store: Store<{ list: Task[] }>) {
+    this.list$.subscribe(res => {})
```

Comment:
what does this do ?

Reply:
was just for a debug at some point, I will remove